viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -396,15 +398,9 @@ void SLAMServiceImpl::GetLatestSampledPointCloudMapString(
             float z_pos = 0;  // Z is 0 in 2D SLAM
 
             // Add point to buffer
-            Eigen::Vector3d map_point(x_pos, y_pos, z_pos);
-            Eigen::Vector3d rotated_map_point = pcdRotation * map_point;
-
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.x());
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.y());
-            viam::utils::writeFloatToBufferInBytes(pcd_data,
-                                                   rotated_map_point.z());
+            viam::utils::writeFloatToBufferInBytes(pcd_data, x_pos);
+            viam::utils::writeFloatToBufferInBytes(pcd_data, x_pos);
```

Comment:
x_pos is written twice

Reply:
Nice catch! That resolves the issue around # of map points decreasing in the integration test!